src/main/java/WorkShop01.kt

```diff
@@ -2,36 +2,47 @@ object WorkShop01 {
 
     fun numericValues() {
         //TODO 1.1. Необходимо создать целочисленную переменную, которая может вместить 10^9
+        var l1: Long = 1000000000
 
         //TODO 1.2. Необходимо создать целочисленную переменную, которая может вместить 10^10
+        var l2: Long = 10000000000
     }
 
     fun printRealOnePrecisionValue() {
         //TODO 2.1. Необходимо создать вещественную переменную с одинарной точности и инициализировать её значением 0.0F
+        val f1 = 0.0F
 
         //TODO 2.2. Добавить к созданной переменной число 0.1111111111111111F 2 раза
+        val f2: Float = f1 + (2 * 0.1111111111111111F)
 
         //TODO 2.3. Раскомментировать блок кода ниже, который с распечатывает число. В блоке кода указать имя переменной
-        //println("Результат сложения для числа с одинарной точностью равен - " + ИМЯ_ПЕРЕМЕННОЙ)
+        println("Результат сложения для числа с одинарной точностью равен - " + f2)
 
         //TODO 2.4. Запустить функцию main (в ней стоит метка TODO 2.4)
         //TODO 2.5. Сравнить результат с ожидаемым 0.2222222222222222F
+        //Результат - 0.22222222
     }
 
     fun printRealTwoPrecisionValue() {
         //TODO 3.1. Необходимо создать вещественную переменную с двойной точности и инициализировать её значением 0.0
+        val d1: Double = 0.0
 
         //TODO 3.2. Добавить к созданной переменной число 0.1111111111111111 2 раза
+        val d2: Double = d1 + (2 * 0.1111111111111111)
 
         //TODO 3.3. Раскомментировать блок кода ниже, который с распечатывает число. В блоке кода указать имя переменной
-        //println("Результат сложения для числа с двойной точностью равен - " + ИМЯ_ПЕРЕМЕННОЙ)
+        println("Результат сложения для числа с двойной точностью равен - " + d2)
 
         //TODO 3.4. Запустить функцию main (в ней стоит метка TODO 3.4)
         //TODO 3.5. Сравнить результат с ожидаемым 0.2222222222222222
+        //Результат - 0.2222222222222222
     }
 
     //TODO 4. Написать функцию (compareTwoStrings), которая принимает на вход две строки и возвращает результат сравнения
     //Проверить функцию можно с помощью тестов, которые лежат в директории test/java (Имя класса - WorkShop01P4Test)
+    fun compareTwoStrings(str1: String, str2: String): Boolean {
```

Comment:
```suggestion
    fun compareTwoStrings(str1: String, str2: String): Boolean = str1 < str2
```
Мы можем использовать сокращенную запись, так как в теле функции у нас всего одно выражение